WORKSPACE.yaml

```diff
@@ -8,6 +8,7 @@ defaultArgs:
   publishToJBMarketplace: true
   localAppVersion: unknown
   codeCommit: 239fa626a44652cebf293a994a3330e90e873daa
+  codeVersion: 1.72.2
```

Comment:
Note another place to change for code image build. XD

Reply:
Can't we get this from code build? instead of changing codes?

Reply:
Unfortunately image label cannot be computed during Docke build. We have to provide it via arg

Reply:
Why not do it like `code-nightly` do in this PR https://github.com/gitpod-io/gitpod/pull/14055/files#diff-3a4804782c8876a5e55d41725ccd6494495fcd585f75559bf4e1953ba8db5997R36-R37?
```
codeVersion=$(curl https://raw.githubusercontent.com/gitpod-io/openvscode-server/$headCommit/package.json | jq .version)
leeway build -Dversion=nightly -DimageRepoBase=eu.gcr.io/gitpod-core-dev/build -DcodeCommit=$headCommit -DcodeVersion=$codeVersion -DcodeQuality=insider .:docker
```

Reply:
How would you do it in WORKSPACE.yaml for stable? Please feel free to send a PR against this PR.

Reply:
I agree with @mustard-mh this just complicates things unnecessarily, doesn't leeway have a `--label` option like docker? maybe create a PR in leeway repo to support it.

Reply:
@akosyakov @filiptronicek there's `docker-build-options` in leeway

Reply:
This is the same as the implementation [here](https://github.com/gitpod-io/gitpod/pull/14055/files#diff-0b0745a45358607832712b3bf11900bda7b207effc15bddec5e1566da7e45e80R21), we have no way to get the version at docker build time (we checkout openvscode-server only at docker build time)

Although leeway can't do this, we can use the `github raw file api` to get the version number and pass it to leeway before the werft job starts leeway. 
i.e. here https://github.com/gitpod-io/gitpod/blob/cb831115e6f30d21f644d1066588f71de62cd6bd/.werft/jobs/build/build-and-publish.ts#L44-L57

But we can merge this PR first, and do a follow up PR

Reply:
I am doing rebase this PR to main, and build vscode image

Reply:
@mustard-mh what do you mean just pass the version like this `--docker-build-options="label=foo,label=bar"` in `code-nightly.yml`

Reply:
OK build success, let's we merge it, and change `code` stable image in next PR